action/protocol/poll/staking_committee.go

```diff
@@ -34,8 +34,11 @@ import (
 	"github.com/iotexproject/iotex-core/state"
 )
 
-var nativeStakingContractCreator = address.ZeroAddress
-var nativeStakingContractNonce = uint64(0)
+var (
+	nativeStakingContractCreator = address.ZeroAddress
+	nativeStakingContractNonce   = uint64(0)
+	nativeStakingContractHash, _ = hash.HexStringToHash256("000000000000006e61746976655374616b696e67436f6e747261637448617368")
```

Comment:
why do we need a magic string?

Reply:
like i said, the current code gives an all-0 hash